Content/Serializer/MediaSerializerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\HeadlessBundle\Content\Serializer;
+
+use JMS\Serializer\SerializationContext;
+use Sulu\Bundle\MediaBundle\Api\Media;
+
+interface MediaSerializerInterface
+{
+    /**
+     * @return mixed[]
+     */
+    public function serialize(Media $media, ?SerializationContext $context = null): array;
```

Comment:
Should be `MediaInterface`

Reply:
no, the serializer expect an `ApiEntitiy` instance at the moment: https://github.com/sulu/SuluHeadlessBundle/issues/21